09_tabular.ipynb

```diff
@@ -40,14 +40,14 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In tabular data some columns contain numerical data, like \"age\", others contain string values, like \"sex\". The numerical data can be directly fed to the model (with some optional preprocessing), but other columns need to be converted to numbers. Since the values in those correspond to different categories, we often call these type of variables *categorical variables*. The first type are called *continuous variables*."
+    "Tabular modelling takes data in the form of a table (like a spreadsheet or CSV--comma separated values). The objective is to predict the value in one column, based on the values in the other columns. In tabular data some columns, like \"age\", contain numerical data, while others contain string values, like \"sex\". The numerical data can be directly fed to the model (with some optional preprocessing), but other columns need to be converted to numbers. Since the values in those correspond to different categories, we often call these type of variables *categorical variables*. The first type are called *continuous variables*."
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "> jargon: Continuous and categorical variables: \"Continuous variables\" are numerical data, such as \"age\" can be directly fed to the model, since you can add and multiply them directly. \"Categorical variables\" contain a number of discrete levels, such as \"movie id\", for which addition and multiplication don't have meaning (even if they're stored as numbers)."
+    "> jargon: Continuous and categorical variables: \"Continuous variables\" are numerical data and can be directly fed to the model, since you can add and multiply them, e.g. \"age\". \"Categorical variables\" contain a number of discrete levels, such as \"movie id\", for which addition and multiplication don't have meaning (even if they're stored as numbers)."
```

Comment:
Ordinal is used further down and it seemed to jar when I got there. Its nice to keep here about the 2 simple categories, but I also wonder if a new reader would be helped with that definition and something about things like North South East West vs compass headings, or categories like small medium large when they are here?

Reply:
Probably best to introduce things slowly